docs/src/components/DocApiEntry.js

```diff
@@ -78,10 +79,15 @@ function getExtendedNameDiv (label, level, type, required, addedIn) {
   const suffix = `${type ? ` : ${type}` : ''}${required ? ' - required!' : ''}`
 
   const child = [
-    h(QBadge, {
-      color: NAME_PROP_COLOR[ level ],
+    h(QChip, {
```

Comment:
Seems like you're trying to force a QChip to behave as a QBadge here
Is it because QBadge won't allow you to react to click events?

Reply:
Yup, that was the only reason.

Reply:
`QBadge` click event seems to work correctly: https://codepen.io/yusufkandemir/pen/BarzojJ?editors=101
I wonder what's wrong.

Some random thought: `QChip` sounds somewhat more semantical than `QBadge` in this context.

Reply:
Yeah, that was my same doubt, AFAIK you can just listen to click even on a `QBadge`, to keep the code consistent and avoid forcing a `QChip` to look like a `QBadge`

Reply:
I'm gonna take another look at this and report back.

Reply:
I've made some mistakes first time, as `QBadge` is now behaving perfectly on listening to `click` event.